docs/authorization-introduction.md

```diff
@@ -0,0 +1,96 @@
+# Authorization Introduction
+
+This document provides a general overview on what are the terms used in the Udaru authorization system and describes the structure and functionality of the authorization application.
+
+The Udaru authorization system has two main functional components:
+- Using the PBAC authorization mechanism for internal use to authorize the system internal parts against itself,
+- Provide PBAC authorization service for external applications.
+
+## Authorization elements
+
+The access control management evolved from the need to mitigate the risk of unauthorized access to data.
+
+Several control models were built to approach the various access control needs: Access Control Lists, Role Based Access Control, Attribute Based Access Control, Policy Based Access Control, Risk Adaptive Access Control.
+
+Udaru relies on [iam-js][] that is based on [node-pbac][] module, module that is inspired and mostly compatible with the [AWS IAM][] policy engine.
+
+A policy is a document that describes what type of access the policy owner has to a resource or a group of resources. The policy engine evaluates all the policies attached to a user (policies attached to the user, the teams(s) that the user belong to and the organization the user belongs to) and decides if the user has access to perform a certain action over a certain resource.
```

Comment:
```
A user can have policies attached to himself, inherit them from his team(s) and organization.

Given the policies, the action to perform and a resource, the engine return true if the action is authorized, false otherwise.
```

How does that sound?